yeatmanlab/roar-dashboard
# Export combination of Score and Progress Report

## Proposed changes

This PR export score reports into a single function that handles exporting for selected rows, the entire report, and combined reports.

**Key Updates:**
Unified Export Function: The `export-selected`,` export-whole`, and `export-combined` functionalities are now managed by a single function, simplifying the export logic and reducing code duplication.

**Steps to Review:**
- Navigate to the Score Report of an admin.
- Select "Export Combined Reports" to generate the CSV.
- Verify that the exported report is formatted correctly and includes the expected data for both selected rows and the entire report as applicable.


## Types of changes

What types of changes does this pull request introduce?




## Checklist




## Justification of missing checklist items



## Further comments
